prime-router/src/main/kotlin/DocumentationFactory.kt

```diff
@@ -93,10 +96,25 @@ ${element.documentation}
     // gets the documentation
     fun getSchemaDocumentation(schema: Schema): String {
         val sb = StringBuilder()
+        var elementName =
+            if (schema.trackingElement.isNullOrEmpty()) {
+                logger.warn("Schema ${schema.name}: TrackingElement is empty")
+                "none"
+            } else {
+                schema.findElement(schema.trackingElement)?.csvFields?.get(0)?.name.toString()
+            }
+        var extendName = if (schema.extends.isNullOrBlank()) "none" else schema.extends.replace('/', '-')
+        var schemaTrackingName = if (schema.trackingElement.isNullOrBlank()) "none" else schema.trackingElement
+        var schemabaseOn = if (schema.basedOn.isNullOrBlank()) "none" else schema.basedOn
+        var schemaExtends = if (schema.extends.isNullOrBlank()) "none" else schema.extends
 
         sb.appendLine(
             """
 ### Schema:         ${schema.name}
+### Topic:          ${schema.topic}
+### Tracking Element: $elementName ($schemaTrackingName)
+### Base On: [$schemabaseOn](./$schemabaseOn.md)
+### Extends: [$schemaExtends](./$extendName.md)
```

Comment:
Add something to detect when these fields don't have a true link.  Don't want to see None with a link that goes nowhere.

Reply:
Done.